openmc/element.py

```diff
@@ -58,7 +58,7 @@ def __ne__(self, other):
         return not self == other
 
     def __hash__(self):
-        return hash((self._name, self._xs))
+        return hash(str(self))
```

Comment:
I think `hash(repr(self))` would be more explicit. Here you're relying on the fact that `__str__` defaults to `__repr__`.